plugin-server/tests/worker/ingestion/person-state.test.ts

```
@@ -2117,3 +2157,488 @@ describe('PersonState.update()', () => {
         })
     })
 })
+
+describe('person id overrides', () => {
+    // For some reason these tests failed if I ran them with a hub shared
+    // with other tests, so I'm creating a new hub for each test.
+    let hub: Hub
+    let closeHub: () => Promise<void>
+    let teamId: number
+    const timestamp = DateTime.fromISO('2020-01-01T12:00:05.200Z').toUTC()
+
+    beforeAll(async () => {
+        ;[hub, closeHub] = await createHub({})
+    })
+
+    beforeEach(async () => {
+        const organizationId = await createOrganization(hub.db.postgres)
+        teamId = await createTeam(hub.db.postgres, organizationId)
+        // jest.useFakeTimers({ advanceTimers: 50 })
+    })
+
+    afterEach(() => {
+        jest.clearAllTimers()
+    })
+
+    afterAll(async () => {
+        await closeHub()
+    })
+
+    async function updatePersonStateFromEvent(event: Partial<PluginEvent>, ts = '', mergeAttempts = 3) {
```

lets please default to 1, so it's easier to debug and fails faster